start/pom.xml

```diff
@@ -26,6 +26,10 @@
   <name>Apache Accumulo Start</name>
   <description>A library for launching Apache Accumulo services.</description>
   <dependencies>
+    <dependency>
```

Comment:
I suggested using `@VisibleForTesting` annotation, I did not realize it would add this Dep.  I don't think we should add this because we want to keep the deps of start minimal. I can remove this when I merge.